libraries/AP_HAL_ChibiOS/hwdef/f103-QiotekPeriph/hwdef.dat

```diff
@@ -0,0 +1,24 @@
+include ../f103-periph/hwdef.dat
+
+define CAN_APP_NODE_NAME "org.ardupilot.f103Qiotek_Periph"
+
+# use DNA
+define HAL_CAN_DEFAULT_NODE_ID 0
+
+undef  HAL_COMPASS_MAX_SENSORS 1
+define define HAL_COMPASS_MAX_SENSORS 2
```

Comment:
Define define? 
I don't think you the undef line too, but let's wait for someone more qualified to validate

Reply:
good catch Pierre, I didn't even notice!

Reply:
good catch Pierre @khancyr